### DISCUSSION TITLE
Env Variables Not Loaded When Deployed To Cloudflare Pages

### DISCUSSION THREAD

**Discussion by gaurishhs** [webpack/webpack#15091] [General]
# Bug report

**What is the current behavior?**

- Env vars arent loaded when deployed to Cloudflare pages
- I have a repository using [Quasar](https://quasar.dev) and as stated in their Docs [Here](https://quasar.dev/quasar-cli/handling-process-env#using-dotenv) I have imported dotenv and provided that to webpack build, Now when i run it locally it does load env and process isnt undefined whereas when deployed to cloudflare pages, process becomes undefined.
- **If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**
- Env variables should have loaded and process shouldnt have been undefined

**Other relevant information:**
webpack version:  latest
Node.js version: v16.13.0
Operating System: MacOS Intel 2017 version
Additional tools: Quasar,VueJS,Cloudflare

**Comment by alexander-akait** [comment]
What is env variables? Please clarify

**Comment by gaurishhs** [comment]
.env vars?

**Reply by alexander-akait** [reply]
We don't support env vars, you need setup https://github.com/motdotla/dotenv to use it in application
